tools/cldr/.gitignore

```diff
@@ -3,5 +3,5 @@
 !/lib/README.txt
 
 # Ignore the default Maven target directory.
-/target
```

Comment:
Good pont. Can you also add any occurrence of directory /.idea/ here?
It's what IntelliJ uses.

Reply:
Where do these directories appear? If I make it "/.idea", I think it means only ".idea" in the same directory as the gitignore file. If I make it ".idea", it's also in the subdirectories.

Reply:
Added `.idea/` in the top-level .gitignore file, next to `.vscode/`.